dm/dm/dm-ansible/scripts/DM-Monitor-Professional.json

```diff
@@ -5088,7 +5088,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "dm_relay_binlog_file{instance=~\"$instance\", node=\"master\"} - ON(instance, job) dm_relay_binlog_file{instance=~\"$instance\", node=\"relay\"}",
+          "expr": "dm_relay_binlog_file{instance=~\"$instance\", node=\"master\"} - ON(instance) dm_relay_binlog_file{instance=~\"$instance\", node=\"relay\"}",
```

Comment:
preview [link](http://172.16.5.177:3000/d/Ja68YqRnzxxxxxxx/dm-monitor-professional-fixed?editPanel=42&orgId=1&from=now-5m&to=now&var-task=All&var-source=mysql-replica-2-001&var-instance=172.16.5.177:8101&var-queueNo=All&var-type=All&var-stage=All&var-target_schema=All&var-target_table=All&var-interval=$__auto_interval_interval&var-worker=All)